source/dub/generators/build.d

```diff
@@ -579,7 +579,7 @@ class BuildGenerator : ProjectGenerator {
 			settings.compiler.invoke(buildsettings, settings.platform, settings.compileCallback);
 
 			if (generate_binary) {
-				logInfo("Linking", Color.light_green, "%s", buildsettings.targetName.color(Mode.bold));
+				logInfo("Linking", Color.light_green, "%s => %s", buildsettings.targetName.color(Mode.bold), buildsettings.getTargetPath(settings));
```

Comment:
what do others think about this CLI output change? @Geod24 @thewilsonator ?

The output path is for example `/tmp/.dub/build/issue2051-test-application-unittest-linux.posix-x86_64-dmd_v2.100.0-0AC041139053DBA3B189AED31C7F8A27221CA401DEE41BA7B05C3F5F74C4D4BE/issue2051-test-application`, possibly even longer for longer project folder names in local temp cache.

I don't think this is really useful most of the time, should we only show this when having the --temp-build option used?

Reply:
yes ... that would be my preference as well! normally its not really needed!